src/GLideNUI-wtl/Settings.cpp

```diff
@@ -229,6 +232,9 @@ void writeSettings(const char * _strIniFolder)
 		settings.setValue("enableLOD", config.generalEmulation.enableLOD);
 		settings.setValue("enableHWLighting", config.generalEmulation.enableHWLighting);
 		settings.setValue("enableShadersStorage", config.generalEmulation.enableShadersStorage);
+		settings.setValue("enableLegacyBlending", config.generalEmulation.enableLegacyBlending);		 //ini only
+		settings.setValue("enableHybridFilter", config.generalEmulation.enableHybridFilter);			 //ini only
+		settings.setValue("enableFragmentDepthWrite", config.generalEmulation.enableFragmentDepthWrite); //ini only
```

Comment:
I think this section is useless since these settings have no GUI support and in fact are read-only.
The same is for the second file.

Reply:
I could remove those sections but then these settings won't be made visible in the ini, you would have to type them manually. Personally I think that's inconvenient as you'd need to remember the exact name/spelling of these settings, check them in the source code. While in the current code they get written with the default value and do no harm. If you change the value in the ini the new value is kept. So I don't understand what you mean by "read-only" in this case

Reply:
> I could remove those sections but then these settings won't be made visible in the ini, you would have to type them manually. 

Yes, you're right.